core-feature-pack/src/main/resources/configuration/domain/subsystems.xml

```diff
@@ -5,5 +5,6 @@
       <subsystem>logging.xml</subsystem>
       <subsystem>jmx.xml</subsystem>
       <subsystem>request-controller.xml</subsystem>
+      <subsystem>core-management.xml</subsystem>
```

Comment:
Please move this below logging. Order should be "logging" then the others in alphabetical order by subsystem name.

Having "logging" first no longer has any justification but it's a kind of tradition now. :)